src/plugins/foxPage/hooks/getGovernanceData.test.ts

```diff
@@ -0,0 +1,201 @@
+import { parseGovernanceData } from './getGovernanceData'
+
+const EMPTY_RESULTS_PROPOSAL = {
```

Comment:
I'd love to see this typed if it's feasible - we have a bad habit of letting un-typed test data decay. I want to force us to update it as types change.

Reply:
Typed in 03be9dcbb0216be02c732d5cbef2058a0dff9517 and in the process, made the test data non-exhaustive to match the types